proposals/partial-type-inference.md

```diff
@@ -0,0 +1,743 @@
+# Partial type inference
+
+* [x] Proposed
+* [ ] Prototype
+* [ ] Implementation
+* [ ] Specification
+
+> Note: This proposal was created because of championed [Partial type inference](https://github.com/dotnet/csharplang/issues/1349). It is a continuation of the proposed second version published in [csharplang/discussions/7467](https://github.com/dotnet/csharplang/discussions/7467) and the first version published in [csharplang/discussions/7286](https://github.com/dotnet/csharplang/discussions/7286)
+
+## Summary
+[summary]: #summary
+
+Partial type inference introduces a syntax skipping obvious type arguments in the argument list of
```

Comment:
>obvious [](http://example.com/codeflow?start=52&length=7)

nit: instead of "obvious" consider "inferrable" (also applies below)

Reply:
Should be fixed by commit [33b71f0](https://github.com/dotnet/csharplang/pull/7582/commits/33b71f0c489eaa03018d18a9cabf8b2cdf74a43a)